misc/python/materialize/mzcompose.py

```diff
@@ -192,7 +192,7 @@ def __init__(self, repo: mzbuild.Repository, name: str):
             compose = yaml.safe_load(f)
 
         # Stash away sub workflows so that we can load them with the correct environment variables
-        self.workflows = compose.pop("mzworkflows", None)
+        self.yaml_workflows = compose.pop("mzworkflows", None)
```

Comment:
`self.workflows` was not a dict of `Workflow` objects, it was just a dict of parsed JSON fragments. So I renamed it accordingly. In the current `mzcompose.py`  architecture there is no unified dict of `Workflow` objects ... they are all instantiated on demand by `get_workflow`.